aptible/database.go

```diff
@@ -137,6 +137,25 @@ func (c *Client) WaitForOperation(op_id int64) error {
 		status = *op.Payload.Status
 		fmt.Println("Still creating...")
 	}
+	fmt.Println("Done!")
 
 	return nil
 }
+
+// Gets database with specific handle.
+func (c *Client) GetDatabaseFromHandle(env_id int64, handle string) (*models.InlineResponse20014EmbeddedDatabases, error) {
+	params := operations.NewGetAccountsAccountIDDatabasesParams().WithAccountID(env_id)
+	resp, err := c.Client.Operations.GetAccountsAccountIDDatabases(params, c.Token)
+	if err != nil {
+		return nil, err
+	}
+
+	databases := resp.Payload.Embedded.Databases
```

Comment:
Are you planning to handle paging of these results in a follow-up PR?

Reply:
Yep, I'll address that in a PR with getting the latest provision operation on a database.